src/object/object_representation.c

```diff
@@ -8706,6 +8705,7 @@ or_get_json_validator (OR_BUF * buf, REFPTR (JSON_VALIDATOR, validator))
       if (rc != NO_ERROR)
 	{
 	  assert (validator == NULL);
+	  /* TODO: er_set a proper error */
```

Comment:
please set a proper error.

Reply:
A proper error is set in db_json_load_validator/validator->load when there is a parse error. I added ASSERT_ERROR at 8710 to make sure an error is set.